test/quantization/fx/test_quantize_fx.py

```diff
@@ -5479,6 +5479,54 @@ def forward(self, x):
             self.checkGraphModuleNodes(m, expected_node_occurrence=node_occurrence)
             self.checkGraphModuleNodes(m_ref, expected_node_occurrence=node_occurrence_ref)
 
+    def test_match_pattern_with_multiple_args(self):
+        """ Test that we can match a pattern that has multiple arguments
```

Comment:
what does "multiple arguments" mean?

Reply:
this means `reshape` has multiple arguments, also added in the docstring